sdk/appconfiguration/ci.yml

```diff
@@ -28,6 +28,7 @@ pr:
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
+    TestProxy: true
```

Comment:
This will get things working in playback test pipelines, but we'll also need to add `AZURE_TEST_RUN_LIVE` and `AZURE_SKIP_LIVE_RECORDING` per the [pipeline configuration instructions](https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/test_proxy_migration_guide.md#enable-the-test-proxy-in-pipelines) to cover live tests